ibis/formats/__init__.py

```diff
@@ -56,7 +58,7 @@ def to_ibis(cls, typ: T, nullable: bool = True) -> DataType:
         raise NotImplementedError
 
     @classmethod
-    def from_string(cls, text: str, nullable: bool = True) -> DataType:
+    def from_string(cls, text: str, nullable: bool | None = None) -> DataType:
```

Comment:
The subclasses SqlglotType and PostgresType already use this signature. So this is more of a bugfix to bring this ABC into compliance with how we actually are using them.